spec/spec_helper.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
-require "sidekiq/poison/pill/remedy"
+require_relative '../lib/sidekiq_poison_pill_remedy'
```

Comment:
it should be at most required `sidekiq-poison-pill-remedy`,  `require_relative` is rarely ever needed for basic  things in gems